README.md

```diff
@@ -57,30 +57,25 @@ All the commands in this section assume that you are in the
 
 ### Running a local dev server
 
-This is useful for developing backend functionality locally. This
-local dev server can be connected to the frontend applications by
-running `dart dev/deploy.dart --project test --version test`, and
-answering `N` when asked about deploying to App Engine. This will
-build the frontend files and copy them to the directory from which the
-server will serve them.
-
-Set the environment variables `GCLOUD_PROJECT` and `GCLOUD_KEY`.
-Running `dart bin/server.dart` will give more explanation on what
-these values should be. You should also set `COCOON_USE_IN_MEMORY_CACHE`
-to `true` as you typically don't have access to the remote redis
-instance during local development.
+```sh
+$ export GOOGLE_CLOUD_PROJECT=flutter-dashboard-dev # or flutter-dashboard for prod data
+$ export GCLOUD_KEY=#your_secret # Required for reading/writing from Google Cloud
+$ export COCOON_USE_IN_MEMORY_CACHE=true # Use an in mememory cache locally instead of redis to prevent corruption
+$ dart bin/server.dart
+```
+This will output `Serving requests at 0.0.0.0:8080` indicating the server is working.
```

Comment:
After chatting with @keyonghan , it's not possible anymore to run the server locally.

Reply:
Thanks, added a note saying this is only for legacy users.